packages/amplication-server/src/core/assistant/assistant.service.ts

```diff
@@ -35,6 +38,13 @@ enum EnumAssistantFunctions {
   GetProjectPendingChanges = "getProjectPendingChanges",
   GetPlugins = "getPlugins",
   InstallPlugins = "installPlugins",
+  GetServiceModules = "getServiceModules",
+  CreateModule = "createModule",
+  GetModuleDtosAndEnums = "getModuleDtosAndEnums",
+  CreateModuleDto = "createModuleDto",
+  // CreateModuleEnum = "createModuleEnum",
+  // GetModuleActions = "getModuleActions",
+  // CreateModuleAction = "createModuleAction",
```

Comment:
Are they missed or not yet implemented? 
If the second, a better approach is to implement a `throw NotImplemented` exception instead.